crates/apps/Cargo.toml

```diff
@@ -146,23 +143,22 @@ tendermint-rpc.workspace = true
 thiserror.workspace = true
 tokio = {workspace = true, features = ["full"]}
 toml.workspace = true
-tonic.workspace = true
 tower-abci.workspace = true
 tower.workspace = true
 tracing-appender.workspace = true
 tracing-log.workspace = true
 tracing-subscriber = { workspace = true, features = ["std", "json", "ansi", "tracing-log"]}
 tracing.workspace = true
-winapi.workspace = true
```

Comment:
this is used with conditional compilation - we should put in `[target.'cfg(windows)'.dependencies]` instead

Reply:
fixed in a8c3d66b9b557ed4e73156615632905a247a3c04